lib/viewcomponents/io.jsx

```diff
@@ -0,0 +1,66 @@
+var React = require('react');
+var mouseDownDrag = require('../utilities/mouseDownDrag');
+var LabelPosition = require('../enum/ioLabelPosition');
+
+var ioRadius = 4;
+/*
+props:
+- isInput {Boolean}
+- label {String}
+- x {Number}
+- y {Number}
+- size (radius) {Number}
+- labelPosition
+ */
```

Comment:
There is a way to specify property signatures. http://facebook.github.io/react/docs/reusable-components.html#prop-validation